### PR TITLE
suppressing author name on the disclosure page on hotjar screenshot

### LINKED ISSUE #1567: Suppress author name in Data Disclosure form for HotJar recordings
Further to the work to anonymise HotJar screen recordings, we should also suppress the author name in the Data Disclosure form:

screen shot 2019-02-22 at 13 15 00

### PR DESCRIPTION
**Pull request by jyenes**
suppress name on disclosure page, on hotjar screenshots,

closes: #1567 

- [ ] Unit / Integration tests
- [x] Browser tests

### CODE REVIEW

**Review by diversemix — APPROVED** [review]

File: test/hotjar.browser.js [hunk 1]
```diff
@@ -41,3 +62,129 @@ test('test suppression is in place for the coverletter', async t => {
...
+    .typeText(editors.secondReviewerEmail, 'frances@example.net')
+    .typeText(editors.thirdReviewerName, 'George')
+    .typeText(editors.thirdReviewerEmail, 'george@example.org')
+    .typeText(editors.fourthReviewerName, 'Ayesha')
+    .typeText(editors.fourthReviewerEmail, 'ayesha@example.com')
+    .typeText(editors.fifthReviewerName, 'Sneha')
+    .typeText(editors.fifthReviewerEmail, 'sneha@example.net')
+    .typeText(editors.sixthReviewerName, 'Emily')
+    .typeText(editors.sixthReviewerEmail, 'emily@example.org')
+    .click(wizardStep.next)
```
Comment (hem-brahmbhatt) [inline test/hotjar.browser.js:171]: This is a lot of code we've got to maintain if we make changes to the journey. I'm assuming a lot of it was copy and pasted. It's worth exporting this so we can maintain it in one place.

Comment (jyenes) [reply test/hotjar.browser.js:171]: I know, there is another issue open (#1108 ) to refactor the browser tests I was thinking to pick it up the next one to refactor all this duplicate code.

**Review by hem-brahmbhatt — APPROVED** [review]
